components/dialog/dialog.html

```diff
@@ -12,12 +14,36 @@
         border-radius: var(--radius);
     }
 
+    :host(.fullscreen) {
+        top: 0;
+        left: 0;
+        bottom: 0;
+        right: 0;
+        z-index: 100;
+        translate: unset !important;
+        height: unset !important;
+        width: unset !important;
+    }
+
+    :host(.fullscreen) .resize {
+        display: none;
+    }
+
     header {
         display: flex;
         flex-direction: row;
         align-items: center;
-        padding: var(--padding);
         border-bottom: var(--border);
+        height: 3rem;
```

Comment:
with custom header support do we want to assume height?

Reply:
I can convert the hardcoded values to variables to be changed the 90 % will be the variable value, variable will prevent use of important